### PR TITLE
refactor C# namespaces to map to xmlns URIs

### PR DESCRIPTION
**Pull request by tomjebo**

### CODE REVIEW

File: gen/DocumentFormat.OpenXml.Generator.Linq/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (twsouthwick) [inline gen/DocumentFormat.OpenXml.Generator.Linq/Properties/launchSettings.json:1]: not needed

**Review by twsouthwick — CHANGES_REQUESTED** [review]
Is this the right branch?

File: gen/DocumentFormat.OpenXml.Generator.Linq/Properties/launchSettings.json (hunk 1 above)
Comment (tomjebo) [reply gen/DocumentFormat.OpenXml.Generator.Linq/Properties/launchSettings.json]: I removed it (again)

File: samples/IsolatedStorageExceptionWorkaround/IsolatedStorageExceptionWorkaround - Backup (1).csproj [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (twsouthwick) [inline samples/IsolatedStorageExceptionWorkaround/IsolatedStorageExceptionWorkaround - Backup (1).csproj:1]: no need for backups

**Pull request closed without merging**